doc/content/3.documentation/1.concepts/7.routing-slip.md

```diff
@@ -1,10 +1,4 @@
----
-title: Overview
-description: Distributed transactions using MassTransit Routing Slip
-toc: true
----
-
-# Routing Slip
+# Routing Slips
 
 Developing applications using a distributed, message-based architecture significantly increases the complexity of performing transactions, where an end-to-end set of steps must be completed entirely, or not at all. In an application using an ACID database, this is typically done using SQL transactions, where partial operations are rolled back if the transaction cannot be completed. However, this doesn't scale when the steps being to include dependencies outside of a single database. And in the distributed, *microservices* based architectures, the use of a single ACID database is shrinking to completely non-existent.
```

Comment:
`...when the steps **being** to include...` -> `...when the steps **begin** to include...`